### ISSUE TITLE
AID pool - add/remove pool tokens do not work

### ISSUE THREAD

**Issue by ashachaf** [bancorprotocol/webapp#1558]
**Describe the bug**

trying to add liquidity to this pool and i receive "transaction failed" before it goes to metamask.
https://app.bancor.network/eth/pool/add/0xe3BF775Ec5f4F4dFCbb21194B22be1217b815b1d
![image](https://user-images.githubusercontent.com/40430285/122081535-dba44180-ce07-11eb-9135-e8a45c0cd915.png)

**Comment by Velua** [comment]
Yeah as the error suggests the pool does not support adding liquidity. 

The contract was deployed back in 2018, https://etherscan.io/address/0xb85e52268cbf57b97ae15136aa65d4f567b8107c#readContract
